<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonForm</class>
 <widget class="QWidget" name="PersonForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>人员管理</string>
    </attribute>
    <widget class="QTableView" name="person_TView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delPerson_Btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>去除用户</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeDept_Btn">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>12</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>改变权限</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="Admin_RBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>管理员</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Emp_RBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>员工</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="changePass_Btn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>修改密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPreson_Btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>增加用户</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>会员管理</string>
    </attribute>
    <widget class="QPushButton" name="delVIP_Btn">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>注销会员</string>
     </property>
    </widget>
    <widget class="QTableView" name="VIP_TView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>631</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>消费排序</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortALL">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="sort_Btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>排序</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
